internal/server/models/grant.go

```diff
@@ -11,11 +11,6 @@ const (
 	InfraConnectorRole = "connector"
 )
 
-const (
-	CreatedBySystem = 0
-	CreatedByConfig = 1
```

Comment:
consolidating this under `CreatedBySystem` since the server now loads the internal providers and identities using the same mechanism as config providers/identities